mappings/net/minecraft/world/chunk/ServerChunkProvider.mapping

```diff
@@ -12,13 +13,32 @@ CLASS net/minecraft/class_794 net/minecraft/world/chunk/ServerChunkProvider
 		ARG 1 world
 		ARG 2 chunkWriter
 		ARG 3 chunkProvider
+	METHOD <init> (Lnet/minecraft/class_795;Lnet/minecraft/class_1199;Lnet/minecraft/class_2744;)V
+		ARG 1 world
+		ARG 2 chunkWriter
+		ARG 3 generator
+	METHOD method_12773 (Lnet/minecraft/class_1150;Ljava/lang/String;Lnet/minecraft/class_2552;Z)Lnet/minecraft/class_2552;
+		ARG 1 world
+		ARG 3 pos
+	METHOD method_12774 (Lnet/minecraft/class_1196;)V
+		ARG 1 chunk
+	METHOD method_12777 (II)Lnet/minecraft/class_1196;
+		ARG 1 x
+		ARG 2 z
+	METHOD method_14961 (Lnet/minecraft/class_1150;Ljava/lang/String;Lnet/minecraft/class_2552;)Z
+		ARG 1 world
+		ARG 3 pos
 	METHOD method_2125 unloadAll ()V
 	METHOD method_2127 scheduleUnload (II)V
 		ARG 1 y
 		ARG 2 z
+	METHOD method_3864 (II)Z
+		ARG 1 x
+		ARG 2 z
 	METHOD method_3871 getOrGenerateChunk (II)Lnet/minecraft/class_1196;
 		ARG 1 x
 		ARG 2 z
+	METHOD method_4727 save ()V
```

Comment:
rename this one to `flushChunks` or rename the other implementation in class_1193, class_1214, class_1219, class_476, class_1218 and class_1213 to the same name.

Reply:
done